tensorflow/training/docker/2.2.0/py3/Dockerfile.cpu

```diff
@@ -28,13 +28,13 @@ ARG PIP=pip3
 ARG PYTHON_VERSION=3.7.7
 ARG OPENSSL_VERSION=1.1.1g
 
-ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.2_aws/20200801-180340/cpu/py37/tensorflow_cpu-2.2.0-cp37-cp37m-manylinux2010_x86_64.whl
+ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.2_aws/20200810-220210/cpu/py37/tensorflow_cpu-2.2.0-cp37-cp37m-linux_x86_64.whl
```

Comment:
This needs to be "manylinux2010_x86_64", right?

Reply:
Corrected this.